cmake/msys/msys_helper.sh

```diff
@@ -222,9 +212,8 @@ if [ ! -e "$mingw_root/lib/libgig/libgig.dll.a" ]; then
 	make install
 
 	mv "$mingw_root/lib/bin/libakai-0.dll" "$mingw_root/bin"
-	mv "$mingw_root/lib/bin/libgig-7.dll" "$mingw_root/bin"
 
-	if [ $? -ne 0 ]; then
+	if ! mv "$mingw_root/lib/bin/libgig-7.dll" "$mingw_root/bin"; then
```

Comment:
Sure... we're moving to`set -e` for everything else. :+1:.  We've sort of repurposed a fetch/untar script as a build helper so this file is growing in complexity which can affect how it should respond to errors.

Ideally we'd just move away from this technique and leverage something crowdsourced like `scoop` instead, but we're not there yet.